site/content/docs/user/quick-start.md

```diff
@@ -494,3 +494,18 @@ kind, the Kubernetes cluster itself, etc.
 [customize control plane with kubeadm]: https://kubernetes.io/docs/setup/independent/control-plane-flags/
 [access multiple clusters]: https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/
 [release notes]: https://github.com/kubernetes-sigs/kind/releases
```

Comment:
nit: we should prefer to keep this set of links at the end of the document and insert any new sections before them